.context/tutorials/2026-03-28-cloudfront-redirect-refactor_codex.md

```
@@ -0,0 +1,130 @@
+pnpm --dir infrastructure run synth
+```
+
+Quick verification steps:
+
+1. Rebuild the edge handler.
+
+```bash
+pnpm --dir infrastructure run build:edge
+```
+
+2. Confirm the compiled asset exists.
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)